### DISCUSSION TITLE
internal temperatur

### DISCUSSION THREAD

**Discussion by mikewiebe66** [emsesp/EMS-ESP32#698] [Q&A]
I got one question:
I’m able to set an offset for the internal temperatur of my RC30 but I can’t see an entity with this internal temparatur in gui or homeassistant?
Is it my blindness or isn’t this value available?

**Comment by proddy** [comment]
what is the Product ID for your RC30? We only added the temperature offset to RC30's with a product id of 67

**Reply by mikewiebe66** [reply]
> what is the Product ID for your RC30? We only added the temperature offset to RC30's with a product id of 67

Where can I find the Produkt ID? May I get it from the EMS-ESP GUI?

**Reply by mikewiebe66** [reply]
Found it: Type
Thermostat

Name
RC30

Brand
Device ID
0x10

Product ID
67

Version
02.05

**Comment by MichaelDvP** [comment]
If a thermostat is mounted as roomthermostat the corrected value of internal sensor is shown as roomtemperature. If the thermostat is mounted on boiler the values (corrected/uncorrected) are only published in telegram 0xA3. We show these values for RC35 (because is want to see them for scientific reason), for other thermostats they are not shown.

**Reply by mikewiebe66** [reply]
I got an RC30. Configured as Remotecontrol. Internal Temperature ist shown on the display.
I have a "scientific reason" too. ;-)
RC30 is mounted at the best central place in my living room, so I want to use it's internal temperature to regulate my heatings via HASSIO.

**Reply by MichaelDvP** [reply]
If it is mounted as roomthermostat  the measured value should be shown as roomtemp in the assigned hc. 
But we can add the internal sensors, we skipped to keep json output small, but now we have customizations and people can remove unwanted values. Please check with terminal and `read 10 A3` Post response here, in offset 3/4 should be the internal sensor.

**Reply by mikewiebe66** [reply]
Got it! Marked it as my first favorite. Thanks a lot.
